README.md

```diff
@@ -27,16 +31,18 @@ virtual environment by running
 python -m pip install pandamonium
 ```
 
-### Install the old style of global scripts
+### Install oldschool way
 
-If you want to install the original version of `pandamonium` before it became a
-library and was a set of global level Python scripts you can still do that.
+If you can't get `pip` working you can just put the scripts you need
+in your path. Note that this is **deprecated** (but still works for
```

Comment:
I'm not sure if it is true that this is deprecated. I'm fine with making that so, but at the moment it works just fine the "oldschool way".

Reply:
isn't that what deprecation means? It still works fine, it's just not going to be supported in the future.

Reply:
>isn't that what deprecation means? 

you're right. That is what it means. I think I too often in my head think that "deprecated" means "this is getting removed basically ASAP" which really isn't true at all &mdash; usually it is like a major release or a year down the road.